kafl_fuzzer/common/config.py

```diff
@@ -247,22 +248,31 @@ def _parse_with_config(self, parser):
 
         config = confuse.Configuration('kafl', modname='kafl_fuzzer')
 
-        # check default config search paths
+        # check default config locations
         config.read(defaults=True, user=True)
 
+        # ENV based config
+        if 'KAFL_CONFIG_FILE' in os.environ:
+            try:
+                debug_print(f"Checking configuration at KAFL_CONFIG_FILE: {os.environ['KAFL_CONFIG_FILE']}")
+                config.set_file(os.environ['KAFL_CONFIG_FILE'])
+            except Exception as e:
+                logger.error("Failed to read configuration at $KAFL_CONFIG_FILE:\n" + str(e))
+                sys.exit(1)
+
         # local / workdir config
         workdir_config = os.path.join(os.getcwd(), 'kafl.yaml')
+        debug_print(f"Checking configuration at CWD: {workdir_config}")
         if os.path.exists(workdir_config):
-            config.set_file(workdir_config)
-
-        # ENV based config
-        if 'KAFL_CONFIG_FILE' in os.environ:
-            config.set_file(os.environ['KAFL_CONFIG_FILE'])
+            try:
+                config.set_file(workdir_config)
+            except Exception as e:
```

Comment:
What about removing the if
~~~Python3
            try:
                config.set_file(workdir_config)
            except FileNotFoundError as e:
~~~